packages/manager/modules/billing/src/orders/purchaseOrders/edit-purchase/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "purchaseOrders_edit_page_title": "Modifier la référence interne",
+  "purchaseOrders_form_edit_internal_reference": "Créer votre référence interne",
+  "purchaseOrders_form_edit_purchase_order": "Créer votre purchase order",
+  "purchaseOrders_form_edit_internal_reference_label": "Nom de la référence interne",
+  "purchaseOrders_form_edit_internal_reference_placeholder": "Référence interne",
+  "purchaseOrders_form_edit_internal_reference_sub_info": "La référence ne doit pas excéder 20 caractères",
+  "purchaseOrders_form_edit_purchase_order_label": "Nom du purchase order",
+  "purchaseOrders_form_edit_purchase_order_placeholder": "Purchase order",
```

Comment:
I think better to use FR words.
EN words from my point of view are accepted only for technical stuff.

So better to use `bon de commande`

Reply:
This is currently being considered by the PU
Not blocking for Q&C as it is content

Reply:
I have been confirmed by PU. We keep english term. Purchase Order is a recognised term in the accounting domain (the term is masculine).